test/unit/HTMLBuilder.test.ts

```diff
@@ -28,7 +29,90 @@ describe('HTMLBuilder', () => {
   const falseStyleString = objToCSS(cqlLogicClauseFalseStyle);
   const coverageStyleString = objToCSS(cqlLogicClauseCoveredStyle);
 
+  const singlePopMeasure = <fhir4.Measure>{
+    resourceType: 'Measure',
+    status: 'unknown',
+    group: [
+      {
+        population: [
+          {
+            id: 'initial-population-id',
+            code: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/measure-population',
+                  code: 'initial-population'
+                }
+              ]
+            },
+            criteria: {
+              expression: 'ipp',
+              language: 'text/cql'
+            }
+          }
+        ]
+      }
+    ]
+  };
+  const popRetrieveFuncElm = getELMFixture('elm/PopRetrieveFunc.json');
+  const prfStatementResults = [
+    {
+      libraryName: 'Test',
+      statementName: 'A Function',
+      localId: 'test-id-1',
+      final: FinalResult.FALSE,
+      relevance: Relevance.TRUE,
+      raw: undefined,
+      isFunction: true,
+      pretty: 'FUNCTION'
+    },
+    {
+      libraryName: 'Test',
+      statementName: 'SimpleVSRetrieve',
+      localId: 'test-id-2',
+      final: FinalResult.FALSE,
+      relevance: Relevance.TRUE,
+      raw: false,
+      isFunction: false,
+      pretty: 'FALSE (false)'
+    },
+    {
+      libraryName: 'Test',
+      statementName: 'ipp',
+      localId: 'test-id-3',
+      final: FinalResult.FALSE,
+      relevance: Relevance.TRUE,
+      raw: undefined,
+      isFunction: false,
+      pretty: 'IPP'
+    }
+  ];
+  const prfClauseResults = [
+    {
+      raw: undefined,
+      statementName: 'A Function',
+      libraryName: 'Test',
+      localId: 'test-id-1',
+      final: FinalResult.NA
+    },
+    {
+      raw: undefined,
+      statementName: 'SimpleVSRetrieve',
+      libraryName: 'Test',
+      localId: 'test-id-2',
+      final: FinalResult.TRUE
+    },
+    {
+      raw: undefined,
+      statementName: 'ipp',
+      libraryName: 'Test',
+      localId: 'test-id-3',
+      final: FinalResult.TRUE
+    }
+  ];
+
   beforeEach(() => {
+    simpleMeasure = getJSONFixture('measure/simple-measure.json') as fhir4.Measure;
```

Comment:
does this need to be in a beforeEach? If no properties are ever modified by the tests, we should just be able to load this in once before the `describe` as a const for efficiency

Reply:
Agreed.

Reply:
Agreed!